gradle/cli.gradle

```diff
@@ -0,0 +1,52 @@
+import io.fourfinanceit.setup.Accurest
+import io.fourfinanceit.setup.Custom
+import io.fourfinanceit.setup.Maven
+import io.fourfinanceit.setup.MicroInfra
+import io.fourfinanceit.setup.Options
+
+task setup << {
+    println file('src/main/resources/banner.txt').text
+    Console console = System.console()
+    if (console) {
+        Options options = new Options(maven(console), custom(console), accurest(console), microInfra(console))
+        logger.info "Running setup with [$options]"
+        options.process(project)
+    } else {
+        logger.error "Cannot get console."
+    }
+}
+
+Maven maven(Console console) {
+    console.println(">>> Maven configuration <<<")
+    String passedMavenUser = console.readLine("\n> Please enter the Maven User. Current [${mavenUser}]: ") ?: mavenUser
+    String passedMavenPassword = console.readLine("> Please enter the Maven Password. Current [${mavenPassword}]: ") ?: mavenPassword
```

Comment:
I don't know if we should keep Maven credentials stored in project configuration by default. `~/.gradle/gradle.properties` seems to be a better place.